package.json

```diff
@@ -127,13 +120,12 @@
     "@testing-library/user-event": "14.5.2",
     "@types/jest": "29.5.11",
     "@types/react": "18.2.47",
+    "@types/react-dom": "18.2.18",
```

Comment:
Would get warnings when build since it was not available. Since we actually consume the types for it I added the package.